apps/quotes/serializers.py

```diff
@@ -37,7 +35,7 @@ def get_bite(self, instance: 'Quote') -> str:
         return (
             instance.bite.html
             if instance.bite
-            else truncatechars_html(instance.text, 100)
+            else ""
```

Comment:
I hope this was an intentional change

Reply:
:) yes; the truncated html of `instance.text` is already being set in the `truncated_html` field. I figured there's no need to return the same value twice, and this way we can detect whether a bite exists or not, and possibly use some logic depending on that.